### PR TITLE
Copywriting review

### PR DESCRIPTION
**Pull request by astrojuanlu**

### CODE REVIEW

File: examples/mockups/index.html
Comment (astrojuanlu) [inline examples/mockups/index.html:51]: The H1 in the old site is "Technical documentation lives here", which I quite like. "Continuous documentation for technical projects". This proposal is a little bit more emotional. What do you think?

File: examples/mockups/index.html
Comment (astrojuanlu) [inline examples/mockups/index.html:60]: "Start your docs" sounded weird, hope this sounds better? And also signals that the first action the user will take is "importing" their docs, as per our current UI

File: examples/mockups/index.html
Comment (astrojuanlu) [inline examples/mockups/index.html:149]: Typo

File: examples/mockups/index.html
Comment (astrojuanlu) [inline examples/mockups/index.html:157]: Tried to make this a bit shorter, I think the meaning is retained.

File: examples/mockups/index.html
Comment (astrojuanlu) [inline examples/mockups/index.html:165]: Wrote the actual config file name. Should we point to our docs here? https://docs.readthedocs.io/en/stable/config-file/v2.html

File: examples/mockups/index.html
Comment (astrojuanlu) [inline examples/mockups/index.html:191]: "Outputting" sounded weird, hope this is better?

File: examples/mockups/index.html
Comment (astrojuanlu) [inline examples/mockups/index.html:195]: @agjohnson I know you're not a fan of this slogan but I quite like it! Today is my support day and I received zero requests yet. I think for the majority of our users, "it just works". And it's also a sentence we heard a lot during the customer development interviews. What do you think?

Otherwise, we could maybe replace it by another call for action. "Import your docs now" with a button.

File: examples/mockups/product.html
Comment (astrojuanlu) [inline examples/mockups/product.html:70]: Made these "bullet points" a bit longer and more explicit

File: examples/mockups/index.html
Comment (nienn) [reply examples/mockups/index.html:51]: I like the idea of starting with the "push" action, associating the product to version control right from the first word.

I would also love to have feedback from the rest of the team here. What do you think? This is the first sentence the user will see. If you had 3 seconds to call someone's attention to Read the Docs, what would you say?

File: examples/mockups/index.html
Comment (nienn) [reply examples/mockups/index.html:60]: Love it!

File: examples/mockups/index.html
Comment (nienn) [reply examples/mockups/index.html:157]: It works. I like it.

File: examples/mockups/index.html
Comment (nienn) [reply examples/mockups/index.html:165]: The changes look great.

I'm of two minds about adding the link; in one hand, that resource is not the start point for a new user, so by sending him there, we can be diverging him, on the other hand, if we think that someone comes to this text looking information on resources, than the link should be there.

File: examples/mockups/index.html
Comment (nienn) [reply examples/mockups/index.html:195]: I don't have strong feelings for "it just works", one way or another, but I'd love to have a bold statement here!
We also talked about the importance of having a slogan for the product.

Either way, adding an action button here seems a good idea.

Comment (agjohnson) [reply examples/mockups/index.html:195]: So yeah I'm +1 on a bold statement here. But "it just works" sounds too simplified and vague and a bit handy wavy to me. "It just works" could be applied almost anywhere -- electrical service to my house just works :slightly_smiling_face:. But I *do* like our sentiment behind this statement: users get a lot of value out of minor interaction with us.

I think we can retain the sentiment, and be less vague/generic with this. Clearer language would go a long way here. There were a few suggestions here that I think make sense to explore: focus on what matters: your documentation, spend less time on infrastructure around your documentation, etc. Snappier of course though.

File: examples/mockups/product.html
Comment (agjohnson) [inline examples/mockups/product.html:62]: Pull request is specific to GitHub, but they do not reference it as a proper noun:
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests

```suggestion
                Read the Docs simplifies the entire process of building and deploying developer documentation. With support for Sphinx, Mkdocs, and Jupyter Book, you can integrate your code and user-facing documentation using the same tools. Create beautiful documentation easily with themes, and preview every commit with pull request previews.
```

File: examples/mockups/product.html
Comment (agjohnson) [inline examples/mockups/product.html:77]: :100:

File: examples/mockups/index.html
Comment (agjohnson) [reply examples/mockups/index.html:191]: I like shipping more than outputting. We could also move this around to drop the verb: "We can generate HTML, PDF, and ePUB formats for your project, and will host and manage ..."

File: examples/mockups/index.html
Comment (agjohnson) [reply examples/mockups/index.html:157]: As a step that describes what RTD does, does this step seem necessary to call out?

I feel like steps should be something like:

* You write docs in your tools
* We build HTML/etc docs and host them
* Your project has search, managed versions, private authentication, etc. Emphasis on "you don't have to do anything here, RTD does all of this for you".

I suppose going back a bit, we introduced RTD to the user with "We do the rest" but then haven't really sold the user on what is included in "the rest".

Comment (agjohnson) [reply examples/mockups/index.html:157]: Also though, I suppose this could be tweaked slightly too, to describe *what RTD does* instead of *what the user is doing*. "RTD watches your repository and rebuilds your documentation any time you merge a change" -- though it would probably help to be a bit less technical here.

File: examples/mockups/index.html
Comment (nienn) [reply examples/mockups/index.html:60]: "Get started!" is simple, to the point and works. "Import your docs" had the +1 of telling you something about the process.

By this logic, after reading the issue you linked I would also consider "Add your project" or "Connect your repo". Only, these might not sound as clear as "Get started!"

File: examples/mockups/index.html
Comment (ericholscher) [reply examples/mockups/index.html:51]: > even though we could do more to be more technically correct or communicate what "the rest" is here. 

This is my main feedback, but I couldn't find a way to improve it with a similar number of words. I think the rest of the page will explain "the rest", so this seems fine for a first pass.

File: examples/mockups/index.html
Comment (ericholscher) [reply examples/mockups/index.html:191]: I don't like shipping, it feels pretty jargony. I'd use `creating` or `generating` as suggested.

File: examples/mockups/index.html
Comment (ericholscher) [reply examples/mockups/index.html:195]: `Get your docs online in 5 minutes` or something perhaps? I do agree we should deconstruct what "just works". Meaning it's easy to setup? Doesn't require much maintenance?

File: examples/mockups/product.html
Comment (ericholscher) [inline examples/mockups/product.html:84]: This is a pretty minor feature that many people don't use... I'd probably focus on something else here.

**Review by ericholscher — APPROVED** [review]
I don't have a ton of additional feedback here, I'd say lets just move forward with most of what we have here. I think getting something live is more important than getting it perfect, so let's 🚢 it.

File: examples/mockups/index.html
Comment (humitos) [reply examples/mockups/index.html:51]: What is "the rest"? :) To me, generally speaking, is "build and host". So, what about something that includes those words?

* "Push your docs, we will build and host them nicely"
* "Push your docs, we will host them nicely"

("nicely" is not the exact word that I want, but probably a synonymous of it)

Comment (agjohnson) [reply examples/mockups/index.html:51]: Seems we mostly agree on "the rest" being vague, perhaps it makes sense to continue on and iterate on this later?

To me, this is enough to work with and we can come back to this with more ideas. I actually think keeping the header short and vague works well, especially in combination with supporting elements to help define "the rest". If we want to continue on this path, I have some ideas for supporting elements which we could add later.

File: examples/mockups/index.html
Comment (agjohnson) [reply examples/mockups/index.html:60]: True. Though I just realized that if we want to link to the Sphinx getting started guide, this doesn't align with any call to action that references add an existing documentation project. The getting started guide walks the users through starting a project from scratch, whereas a call to action like "Add your project" is engaging a user that already has a Sphinx project configured.

So I suppose I would vote to keep this pointing to the Sphinx guide, and make the language in the link something like "Get started!" or maybe even language more specific to starting a project from scratch. Targeting this user here is probably the most helpful, as a user familiar with Sphinx already probably doesn't need to be sold on RTD as much.

File: examples/mockups/index.html
Comment (humitos) [reply examples/mockups/index.html:51]: Oh, yes! I don't think we have to block this work on this 👍🏼

**Comment by humitos** [comment]
> I think getting something live is more important than getting it perfect, so let's ship it.

This is my preferred phrase from you 😃

File: examples/mockups/product.html
Comment (humitos) [reply examples/mockups/product.html:84]: If we want to mention this feature, I'd use say something like "Slack notifications on build failures" which is something that everybody understands and it's easier to sell to non-technical users.

File: examples/mockups/index.html
Comment (agjohnson) [reply examples/mockups/index.html:195]: `Get your docs online in 5 minutes` is a great example -- it's more specific and conveys at least some of what we offer. I do think there is something more specific that we'd eventually want here, probably describing automation or ease of doc management, but this is a good starting point.

File: examples/mockups/index.html
Comment (agjohnson) [reply examples/mockups/index.html:165]: I'd agree we're probably distracting the user at this point by pointing them to the docs on our config file -- especially because those docs are *dense* technical docs. For users that aren't familiar with RTD, they can get a sense of things through our guides.

File: examples/mockups/index.html
Comment (agjohnson) [reply examples/mockups/index.html:51]: Cool, it seems we can continue with the suggestion from @astrojuanlu and I've opened https://github.com/readthedocs/site-community/issues/30 to come back to later.

File: examples/mockups/index.html
Comment (agjohnson) [reply examples/mockups/index.html:60]: Hah yeah, I had the same thought on two buttons. This could be something to test later for sure.

For now, sounds like we agree starting at `Get started!` and can tweak this a little bit. Having a link into our Sphinx guide is a great benefit here though, that doc does a great job of highlighting what the user should expect.

```suggestion
                <a href="https://docs.readthedocs.io/page/intro/getting-started-with-sphinx.html" class="ui button large secondary gradient fade">Get started!</a>
```

File: examples/mockups/index.html
Comment (agjohnson) [reply examples/mockups/index.html:191]: This line starts with `generate` already, so `generating` is redundant, but otherwise I prefer this verb. My first vote is still restructuring the sentence to drop the double verbs altogether and make the sentence clearer.

```suggestion
                  We generate HTML, PDF, and ePUB formats for your project, and host and manage documentation versions for you.
```

File: examples/mockups/index.html
Comment (astrojuanlu) [reply examples/mockups/index.html:195]: "Get your docs online in 5 minutes" is nice, but doesn't convey "and then forget that we exist" -- although that's admittedly overly optimistic for some use cases, most of our projects do zero maintenance on their docs once they're online, which is what I somehow wanted to communicate here.

File: examples/mockups/product.html
Comment (astrojuanlu) [reply examples/mockups/product.html]: Tweaked this sentence a bit

File: examples/mockups/index.html
Comment (astrojuanlu) [reply examples/mockups/index.html:195]: In any case, changing to `Get your docs online in 5 minutes` which is less contentious.

File: examples/mockups/index.html
Comment (astrojuanlu) [reply examples/mockups/index.html:157]: Using that sentence, we can come back to it later.

**Comment by astrojuanlu** [comment]
Addressed all the comments, there are no outstanding ones ✔️  @nienn feel free to merge this!

**Comment by nienn** [comment]
It seems to me that all the discussions here have been addressed and resolved in a satisfactory manner. 🚢 
There's also several PR's chained in this one waiting for it to be merged and this seems like an improvement over the previous copy, so I'm merging it for now.

We can continue iterating on copy as we go and maybe open separate issues to discuss more complex or controverse copy pieces, so we unblock the template coding from product positioning.

File: examples/mockups/index.html
Comment (agjohnson) [reply examples/mockups/index.html]: We should create issues for anything that we're punting on. I don't want to lose this conversation, these blocks will require some attention.

File: examples/mockups/index.html
Comment (agjohnson) [reply examples/mockups/index.html]: Yeah, I agree here as well, but at least prefer "Get your docs online" over a "It works". I think the above suggestion on focusing on automation or management is the correct focus for this block long term.

File: examples/mockups/index.html
Comment (agjohnson) [reply examples/mockups/index.html]: Opened https://github.com/readthedocs/site-community/issues/32 to continue this discussion, I'd consider this a fairly important change.

**Pull request merged**
